src/components/elements/input/RadioGroupInputOption.tsx

```diff
@@ -80,9 +87,22 @@ const RadioGroupInputOption: React.FC<Props> = ({
         }}
       />
       {helperText && (
-        <CommonHtmlContent variant='body2' sx={{ ml: 4 }} id={descriptionId}>
-          {helperText}
-        </CommonHtmlContent>
+        <FormHelperText
```

Comment:
Let me know what you think about the change to add a <FormHelperText> wrapper around the helper text of radio buttons. I think it's better for helper text to be consistent, but if we do merge this, we may want to re-open Tarrant's SPDAT and specify in the custom html there that those helper texts should continue to use the color black, instead of the default helper text color of grey.

Reply:
There is certainly complexity in maintaining both of these orientations. Moving the conversation [to figma](https://www.figma.com/design/F9EHR2IZYuMAlOJ9aOG2IL?node-id=2867-9260#878133179) to propose simplifying to a single orientation (vertical) to reduce maintenance burden. Maybe you can pull this part out to a different branch while we sort that out?

Reply:
Sure sounds good. https://github.com/greenriver/hmis-frontend/compare/release-125...martha/radio-group-helper-text?expand=1

Reply:
https://github.com/open-path/Green-River/issues/6378